jax/_src/api.py

```diff
@@ -2611,6 +2611,7 @@ def __init__(self, shape, dtype):
 
 
 def checkpoint(fun: Callable, concrete: bool = False, prevent_cse: bool = True,
+               saveable_policy: Optional[Callable[..., bool]] = None,
```

Comment:
I don't really like saveable_policy; would be okay with just policy (but no strong preference)

Reply:
Done.